src/components/home/slider/SliderContent.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {FC} from 'react'
+import {useSlider} from 'src/context/slider'
+
+import {SliderModal} from './SliderModal'
+import * as SliderParent from './style'
+
+interface SliderContentChildProps {
+  currentSlide?: number
+  translate?: number
+}
+
+export const SliderContentChild: FC<SliderContentChildProps> = ({
```

Comment:
It would be best to rename this file to `SliderContentChild.tsx`. It's a good practice to have the same file names as component names.

Reply:
I changed it.